src/stylesheets/components/_contact-panel.scss

```diff
@@ -0,0 +1,24 @@
+.app-c-contact-panel {
+  @include govuk-responsive-margin($govuk-spacing-responsive-6, "top");
+  @include govuk-responsive-padding($govuk-spacing-responsive-3);
+  background-color: $govuk-blue;
+}
+
+.app-c-contact-panel--heading {
+  @extend .govuk-heading-m;
+}
+
+.app-c-contact-panel--body {
+  @extend .govuk-body;
+  margin-bottom: 0;
+}
+
+.app-c-contact-panel--heading,
+.app-c-contact-panel--body,
+a.app-c-contact-panel--link {
+    color: white;
+}
+
+a.app-c-contact-panel--link:hover {
+  color: lighten($govuk-light-blue, 35%);
```

Comment:
Should we be using either a 50% or 25% tint given our guidance on colours?

Reply:
Ideally yes but this is an unusual use case. Lighten 50% would be white.
I've update to 45% to pass AA contrast